src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml

```diff
@@ -17,17 +17,22 @@ limitations under the License.
 <configurator>
   <domain name="CHIP"/>
 
-  <enum name="IdentifyIdentifyType" type="ENUM8">
+  <bitmap name="Feature" type="BITMAP32">
```

Comment:
Please don't add it here; this feature is never actually enabled in Matter, so it's just noise.